articles/libraries/lock/events.md

```diff
@@ -24,15 +24,26 @@ lock.removeListener('signin ready', signinHandlerFn);
 
 - `shown`: Triggers when the Lock early opens.
 - `ready`: Triggers when the Lock is ready for user interaction.
-- `hidden`: Triggers when Lock has hidden.
+- `close`: Triggers when the user manually closes the Lock.
+- `hidden`: Triggers when the Lock has hidden.
 - `signin ready`: Triggers when signin mode view is displayed.
-- `signin submit`: Triggers when signin panel is being submitted (for database and enterprise connections).
+- `signin submit`: Triggers when signin mode is submitted.
+- `signin success`: Triggers when signin has succeeded with no error.
+- `signin error`: Triggers when there's an error on the signin workflow.
 - `signup ready`: Triggers when signup mode is displayed.
+- `signup submit`: Triggers when singup mode is submitted.
+- `signup success`: Triggers when signup was succeeded with no error.
+- `signup error`: Triggers when there's an error on the signup workflow.
 - `reset ready`: Triggers when reset mode is displayed.
+- `reset submit`: Triggers when reset mode is submitted (for database and enterprise connections).

```

Comment:
We don't have reset on database and enterprise connection... probably is redundant.